### PR TITLE
added option for base-href

### PR DESCRIPTION
**Pull request by radicaldrew**
issue when running homer-webapp from a subdirectory such as /homer
assets do not load correctly since base href is /
please review if this is a good option.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,6 +6,10 @@
     <title>HOMER</title>
     <meta http-equiv="Access-Control-Allow-Origin" content="*">
     <meta name="viewport" content="width=device-width, initial-scale=0.8, maximum-scale=1.0, user-scalable=no" />
+    <script>
+       window['base-href'] = window.location.pathname;
+       console.log(window['base-href']);
```
Comment (adubovikov) [inline src/index.html]: oh oh :-)  please no console.log

**Comment by radicaldrew** [comment]
> Thanks @radicaldrew for contributing a solution. Please sign the CLA so the change can be reviewed & merged in.
> 
> EDIT: you were faster than me

**Comment by adubovikov** [comment]
please remove console.log

**Comment by radicaldrew** [comment]
> please remove console.log

Removed console.log   Thanks

**Review by adubovikov — APPROVED** [review]

**Comment by adubovikov** [comment]
@radicaldrew  
unfortunately the patch breaks share link and another components. We changed it a bit and you can provide the path in the index.html right now (static) We are working on a new solution.

**Pull request merged**